src/browser/convert_options.js

```diff
@@ -13,23 +13,20 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-/*
-    src/browser/convert_options.js
- */
 
 // built-in modules
-let fs = require('fs');
-let path = require('path');
+const fs = require('fs');
+const path = require('path');
 
-let ResourceFetcher = require('electron').resourceFetcher;
+const ResourceFetcher = require('electron').resourceFetcher;
 
 // npm modules
-let _ = require('underscore');
+const _ = require('underscore');
 
 // local modules
-let coreState = require('./core_state.js');
-let log = require('./log');
-let regex = require('../common/regex');
+const coreState = require('./core_state.js');
+import * as log from './log';
+import * as regex from '../common/regex';
```

Comment:
Are you suggesting all the modules be imported or just `coreState`? It seems like currently we're only importing modules that have made the TS transition.